client/style.scss

```diff
@@ -111,7 +111,7 @@
 }
 
 .payment-method-details {
-	display: flex;
+	display: inline-flex;
```

Comment:
This is to make sure all items are aligned left, instead of center on the page:

Before:
<img width="373" alt="Screen Shot 2022-07-19 at 09 28 35" src="https://user-images.githubusercontent.com/12542046/179704630-fb5d9a62-e9b4-4f83-be06-16d70af7d44e.png">

After:
<img width="387" alt="Screen Shot 2022-07-19 at 09 28 25" src="https://user-images.githubusercontent.com/12542046/179704651-6834d698-a136-46e3-b49d-726f3ef1f479.png">